datacapture/src/main/java/com/google/android/fhir/datacapture/views/QuestionnaireItemEditTextQuantityViewHolderFactory.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.fhir.datacapture.views
+
+import android.text.InputType
+import com.google.fhir.r4.core.Integer
+import com.google.fhir.r4.core.QuestionnaireResponse
+
+/* Currently inheriting with QuestionnaireItemEditTextViewHolderFactory, but if we want to add drop down
+    then we need to change the parent Factory*/
```

Comment:
nit: use kotlin doc comment format: https://kotlinlang.org/docs/reference/kotlin-doc.html
```suggestion
/**
 * Inherits from QuestionnaireItemEditTextViewHolderFactory as only the numeric part of the quantity is being handled right now.
 * Will use a separate layout to handle the unit in the quantity.
 */
```

Reply:
Done

Reply:
I don't think this is done? Do you need to push new change?

Reply:
Looking at the latest commit this hasn't been updated. Can you do that please? Thanks!

Reply:
Done